### ISSUE TITLE
HDR10Plus on Android+Amlogic - does not play the video (black screen)

### ISSUE THREAD

**Issue by rafi-d** [xbmc/xbmc#19914]
Hi,
With Kodi 19.1/android/Amlogic - I simply cannot manage to play 4K/HDR10plus/X265 video.
Kodi 18 works fine on the exact same box.
All other players - work OK on the same box.
On Windows - both Kodi 18/19 work.
I am expecting HDR->to->SDR from HDR10Plus ( the TV is SDR/FHD.)

### To Reproduce
Just try to play this :
https://rarbgtor.org/torrent/ms6a4ui

Another observation - HDR10 - plays OK with 19.1, with good colors! I wish it was the same with HDR10Plus.

System:
Android 9
Amlogic s905x2
TV: SDR

**Comment by CRoWare** [comment]
Hi,
Same issue for me on Xiaomi Mi Box S (Amlogic s905) for all HDR10+ contents.
Forum thread here: https://forum.kodi.tv/showthread.php?tid=366432

**Comment by stavultras** [comment]
Any updates?

**Comment by CRoWare** [comment]
Yep, purchased a Nokia Streaming Box 8000. Works fine with this one

**Comment by stavultras** [comment]
I had a similar problem yesterday with Vontar X3 (S905X3). Tried 5 different firmware. And a miracle happened - CoreELEC 19.5-Matrix_rc2 worked fine!

**Comment by rafi-d** [comment]
Update:
I found the setting that causes this : (expert) System->videio->allow HW acceleration - MediaCodec (surface)

If you disable it - the video shows up. Just hat this is VERY needed if your CPU is not that strong, and you might see stuttering.. . Any know W/A?
